Description: no Mac OS X  Think Different
  It was a build error
Forwarded: probably should
Author: Geert Stappers <stappers@stappers.nl>
Last-Updated: 2017-05-27

--- a/app/src/processing/app/Base.java
+++ b/app/src/processing/app/Base.java
@@ -49,7 +49,6 @@
 import processing.app.helpers.filefilters.OnlyFilesWithExtension;
 import processing.app.javax.swing.filechooser.FileNameExtensionFilter;
 import processing.app.legacy.PApplet;
-import processing.app.macosx.ThinkDifferent;
 import processing.app.packages.LibraryList;
 import processing.app.packages.UserLibrary;
 import processing.app.syntax.PdeKeywords;
@@ -132,14 +131,6 @@
     }
     System.setProperty("java.net.useSystemProxies", "true");
 
-    if (OSUtils.isMacOS()) {
-      System.setProperty("apple.laf.useScreenMenuBar",
-        String.valueOf(!System.getProperty("os.version").startsWith("10.13")
-          || com.apple.eawt.Application.getApplication().isAboutMenuItemPresent()));
-
-      ThinkDifferent.init();
-    }
-
     try {
       INSTANCE = new Base(args);
     } catch (Throwable e) {
